pkg/controller/account/account_controller.go

```diff
@@ -171,14 +171,30 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, nil
 	}
 
-	// Ignore accounts for which we're (asynchronously) initializing regions -- unless they've been
-	// in that state for too long.
+	// Detect accounts for which we kicked off asynchronous region initialization
```

Comment:
We need to do this before `accountCreatingTooLong`. Otherwise we miss recovering accounts that were creating+initializing for more than 25 minutes but less than 100.